devicon.json

```diff
@@ -536,6 +536,33 @@
         "color": "#002A41",
         "aliases": []
     },
+    {
+        "name": "ballerina",
```

Comment:
Would you please add alternative names (`altnames`) for the icon? Should be something similar to the following.
```json 
        "name": "ballerina",
        "altnames": ["1st", "2nd", "etc."],
```


__
[![sema-logo](https://app.semasoftware.com/img/sema-tray-logo.gif)](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix&nbsp; | &nbsp;**Tags:** Not maintainable